487 docs   expliquer le moteur de jeu (#603)
JeremieLeymarie/story-builder: documentation/docs/Wiki/wiki_creation.mdx

---
sidebar_position: 10
---

# Wiki

Wikis in Story Builder allow you to develop the narration of your story even further by adding lore oriented elements.

> To create a wiki you need to click on the wiki related section in the navbar and then again click the yellow button to create you first wiki :

Reviewer: ```suggestion
To create a wiki you need to click on the wiki related section in the navbar and then again click the yellow button to create you first wiki:
```

<img src="/img/wiki_homepage.png" alt="Wiki homepage" width="500" />

> You then get pop windows with a small form that you need to fill out to create your first wiki.

Reviewer: ```suggestion
You then get a pop-up window with a small form that you need to fill out to create your first wiki.
```

<img src="/img/wiki_form.png" alt="wiki form" width="700" />

> Now that my wiki is created you can take a look at the main page when it is still empty :

Reviewer: ```suggestion
Now that your wiki is created you can take a look at the main page when it is still empty :
```

<img src="/img/wiki.png" alt="Wiki" width="800" />

> From there all you have to do is create your first article with the "create a first article" on the bottom of the page.

Reviewer: ```suggestion
From there all you have to do is create your first article with the "create a first article" button on the bottom of the page.
```

If you click the buton you the following form to create your article :

<img src="/img/wiki_article_form.png" alt="wiki article form" width="800" />

You just need to fill out the form with the required informations and TADA you have created your first article on your newly created wiki :

<img src="/img/wiki_form_creation.png" alt="wiki form creation" width="800" />

<img src="/img/wiki_first_article.png" alt="wiki first article" width="800" />

> You can also create a new category with the sidebar button on the left of your screen :

Reviewer: ```suggestion
You can also create a new category with the sidebar button on the left of your screen:
```

<img src="/img/wiki_tools.png" alt="wiki tools" width="200" />

Creating a new category allows you to organize your wiki with a much better efficiency.

Here we can create a new category entitled "Weapons", a small popup appears where you enter the name of this new category and the color of the associated tag :

<img src="/img/wiki_new_category.png" alt="wiki new category" width="800" />

> Just press enter and congratulations you have created your first wiki !!!

Reviewer: ```suggestion
Just press enter and congratulations you have created your first wiki!!!
```
